python/cudf/cudf/tests/test_copying.py

```diff
@@ -114,7 +114,8 @@ def test_series_setitem_partial_slice_cow_on():
 
         new_slice = actual[2:]
         assert (
-            new_slice._column.base_data._ptr == actual._column.base_data._ptr
+            new_slice._column.base_data._base._ptr
```

Comment:
Maybe once we've merged spilling and cow we can find a cleaner way to test this? Might be nice to add a TODO. I don't have a particular suggestion, but it's a bit unfortunate that our only option right now is to reach multiple times into the internals. Perhaps there isn't really a good alternative though.

Reply:
Could you add a TODO for this @madsbk?

Reply:
done